### PR TITLE
Corrected script for some item (ShadowGear items)

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: 

Data correction for item IDs  24427, 	24428, 	24429, 	24430, 	24431, 	24439, 	24440, 	24441, 	24442, 	24444, 	24445, 	24446, 	24448, 	24450, 	24451, 	24452, 	24453, 	24454, 	24456, 	24457, 	24458, 	24460, 	24462, 	24463, 	24464, 	24465, 	24466, 	24468, 	24469, 	24470, 	24472, 	24474, 	24475, 	24477, 	24481

https://www.divine-pride.net/database/item/24427
https://www.divine-pride.net/database/item/24428
https://www.divine-pride.net/database/item/24429
https://www.divine-pride.net/database/item/24430
https://www.divine-pride.net/database/item/24431
https://www.divine-pride.net/database/item/24439
https://www.divine-pride.net/database/item/24440
https://www.divine-pride.net/database/item/24441
https://www.divine-pride.net/database/item/24442
https://www.divine-pride.net/database/item/24444
https://www.divine-pride.net/database/item/24445
https://www.divine-pride.net/database/item/24446
https://www.divine-pride.net/database/item/24448
https://www.divine-pride.net/database/item/24450
https://www.divine-pride.net/database/item/24451
https://www.divine-pride.net/database/item/24452
https://www.divine-pride.net/database/item/24453
https://www.divine-pride.net/database/item/24454
https://www.divine-pride.net/database/item/24456
https://www.divine-pride.net/database/item/24457
https://www.divine-pride.net/database/item/24458
https://www.divine-pride.net/database/item/24460
https://www.divine-pride.net/database/item/24462
https://www.divine-pride.net/database/item/24463
https://www.divine-pride.net/database/item/24464
https://www.divine-pride.net/database/item/24465
https://www.divine-pride.net/database/item/24466
https://www.divine-pride.net/database/item/24468
https://www.divine-pride.net/database/item/24469
https://www.divine-pride.net/database/item/24470
https://www.divine-pride.net/database/item/24472
https://www.divine-pride.net/database/item/24474
https://www.divine-pride.net/database/item/24475
https://www.divine-pride.net/database/item/24477
https://www.divine-pride.net/database/item/24481

### CODE REVIEW

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -102110,20 +102090,15 @@ Body:
...
-               bonus bSpeedRate,25;
-            }
-         }
+      bonus2 bAddClass,Class_All,1+.@r/2;
+      bonus bMatkRate,1+.@r/2;
+      if (.@r >= 7) {
+         bonus bDef,15;
+         bonus bFlee,15;
       }
+      if (.@r >= 9)
```
Comment (Atemo) [inline db/re/item_db_equip.yml:102100]:
```suggestion
      if (.@r >= 10)
```

File: db/re/item_db_equip.yml [hunk 2]
```diff
@@ -102380,11 +102313,7 @@ Body:
     EquipLevelMin: 99
     Refineable: true
     Script: |
-      bonus2 bSkillAtk,"RA_CLUSTERBOMB",5;
-      .@r = getrefine();
-      if (.@r>=2) {
-         bonus2 bSkillAtk,"RA_CLUSTERBOMB",.@r/2*3;
-      }
+      bonus2 bSkillAtk,"RA_CLUSTERBOMB",5+2*(getrefine()/2);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:102316]:
```suggestion
      bonus2 bSkillAtk,"RA_CLUSTERBOMB",5+3*(getrefine()/2);
```

File: db/re/item_db_equip.yml [hunk 3]
```diff
@@ -102861,7 +102728,7 @@ Body:
     EquipLevelMin: 99
     Refineable: true
     Script: |
-      bonus2 bAddClass,Class_All,2+(getrefine()/3);
+      bonus2 bAddClass,Class_All,3+(getrefine()/3);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:102731]: It was fine according to KRO description
```suggestion
      bonus2 bAddClass,Class_All,2+(getrefine()/3);
```

**Review by Atemo — CHANGES_REQUESTED** [review]

File: db/re/item_db_equip.yml (hunk 2 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:102316]: Thanks

File: db/re/item_db_equip.yml (hunk 3 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:102731]: Thanks

**Review by Atemo — APPROVED** [review]

**Comment by Atemo** [comment]
Thanks!

**Pull request merged**